src/main/java/org/monarchinitiative/fhir2hpo/io/LoincAnnotationParser.java

```diff
@@ -22,9 +23,9 @@
 public class LoincAnnotationParser {
 	private static final Logger logger = LogManager.getLogger();
 
-	public static Map<LoincId, Loinc2HpoAnnotation> parse(File file, Map<TermId, Term> hpoTermMap) throws FileNotFoundException {
+	public static Map<LoincId, DefaultLoinc2HpoAnnotation> parse(File file, Map<TermId, Term> hpoTermMap) throws FileNotFoundException {
 
-		Map<LoincId, Loinc2HpoAnnotation.Builder> builders = new LinkedHashMap<>();
+		Map<LoincId, DefaultLoinc2HpoAnnotation.Builder> builders = new LinkedHashMap<>();
 		BufferedReader reader = new BufferedReader(new FileReader(file));
 		reader.lines().forEach(serialized -> {
 			String[] elements = serialized.split("\\t");
```

Comment:
There are a couple of checks for the length of `elements` below, which would be better using a constant instead of the magic number 13.